presto-product-tests/bin/run_on_docker.sh

```diff
@@ -89,6 +89,15 @@ function stop_docker_compose_containers() {
   echo "Docker compose containers stopped: [$ENVIRONMENT]"
 }
 
+function prefetch_images_silently() {
+  local IMAGES=`environment_compose config | grep 'image:' | awk '{ print $2 }' | sort | uniq`
```

Comment:
Nit (but will leave for now): I like `$(...)` better than backticks since they nest (and is a bit easier to see where they start/end)